examples/jsm/exporters/USDZExporter.js

```diff
@@ -12,7 +12,13 @@ import { decompress } from './../utils/TextureUtils.js';
 
 class USDZExporter {
 
-	async parse( scene, options = {} ) {
+	parse( scene, onDone, onError, options ) {
+
+		this.parseAsync( scene, options ).then( onDone ).catch( onError );
```

Comment:
AFAICS, this isn't a real sync method since you can do the same like with `STLExporter`:
```js
const result = exporter.parse( mesh );
saveString( result, 'box.stl' );
```
I guess this is the reason why `parse()` does not exist so far. The implementation depends on async/await code which is incompatible with our typical `parse()` semantic.

Reply:
Yes, it's not sync - just callback based. Based on [how GLTFExporter does it](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/exporters/GLTFExporter.js#L183).

Reply:
Okay, since `GLTFExporter` uses the same pattern this change is fine.